ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2722,3 +2722,42 @@ def test_numpy_instance_mod__(
         method_flags=method_flags,
         on_device=on_device,
     )
+    
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="ptp",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_axis=-1,
+        max_axis=0,
+        min_num_dims=1,
```

Comment:
Please don't restrict the `min_num_dims` to 1. We need to keep our tests coverage maximum and so test the function with arbitrary dimensions